src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -3,7 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CS1573</NoWarn>
     <ClsCompliant>true</ClsCompliant>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'uapaot'">true</IsPartialFacadeAssembly>
+    <GenrateTypeForwardsSourceCode Condition="'$(TargetGroup)' == 'uapaot'">true</GenrateTypeForwardsSourceCode>
```

Comment:
Typo on the property `GenrateTypeForwardsSourceCode` -> `GenerateTypeForwardsSourceCode`? Is this changing how we declare `PartialFacades`? Will this be the new property?

Reply:
I don't think we should be changing the property that triggers this